SemDiff.Core/GitHub.cs

```diff
@@ -92,6 +94,20 @@ public Task UpdateLimitAsync()
             {
                 EtagNoChanges = headerVal.Single();
             }
+            if (response.Headers.TryGetValues("Link", out headerVal))
+            {
+                var parsedLink = headerVal.First().Split('<', '>','"');
```

Comment:
Would a regular expression be better here? Something like `<(http[^ ]*)>; *rel *= *"next"`?